learn/experimental/autobatching.md

```diff
@@ -0,0 +1,42 @@
+# Auto-batching
+
+If enabled, multiple document addition requests will be automatically combined into a batch and processed together to speed up indexation.
+
+We would like your feedback on this feature! [Join the discussion](https://github.com/meilisearch/meilisearch/discussions/2070).
```

Comment:
Should we have a one-sentence disclaimer about this being experimental and thus liable to crash, burn, and mutate?